### PR TITLE
Fixed Goerli ENS resolving issues on dev and staging

### PR DESCRIPTION
**Pull request by deekshithepns**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "configurations": [
         {
-            "type": "msedge",
+            "type": "chrome",
```
Comment (mishramonalisha76) [inline .vscode/launch.json:4]: any reason for changing this?

File: src/components/chat/w2wChat/searchBar/SearchBar.tsx [hunk 2]
```diff
@@ -117,10 +116,17 @@ const SearchBar = () => {
...
       try {
-        const address = await provider.resolveName(searchedUser);
-        // this ensures address are checksummed
-        ens = ethers.utils.getAddress(address.toLowerCase());
+        // address = await provider.resolveName(searchedUser);
+        // if(!address){
+        //   address = await library.resolveName(searchedUser);
+        // }
+        // // this ensures address are checksummed
+        // ens=ethers.utils.getAddress(address.toLowerCase());
```
Comment (mishramonalisha76) [inline src/components/chat/w2wChat/searchBar/SearchBar.tsx:126]: remove the commented code

File: src/hooks/useResolveEns.ts [hunk 3]
```diff
@@ -1,31 +1,57 @@
...
+//   let checksummedAddress: string;
+//   address = await provider.resolveName(ensName);
+
+//   // this ensures address are checksummed
+//   checksummedAddress = ethers.utils.getAddress(address.toLowerCase());
+
+//   return checksummedAddress;
+// };
 
 export function useResolveEns(username: string): string {
```
Comment (mishramonalisha76) [inline src/hooks/useResolveEns.ts:45]: Can you try with two optional input params, one as address and one as ens, then we can do the jobs for each, as pe which input is provided?

**Review by mishramonalisha76 — CHANGES_REQUESTED** [review]

File: .vscode/launch.json (hunk 1 above)
Comment (mishramonalisha76) [inline .vscode/launch.json:4]: any reason of changing this

**Review by mishramonalisha76 — CHANGES_REQUESTED** [review]

**Review by 0xNilesh — DISMISSED** [review]

**Review by 0xNilesh — APPROVED** [review]

**Review by 0xNilesh — APPROVED** [review]

**Review by mishramonalisha76 — APPROVED** [review]

**Pull request merged**